plugins/foods.py

```diff
@@ -161,3 +169,58 @@ def taco(text, action):
 
     # act out the message
     action(generator.generate_string())
+
+@asyncio.coroutine
+@hook.command
+def beer(text, action):
+    """<user> - give a beer to <user>"""
+    user = text.strip()
+
+    if not is_valid(user):
+        return "I can't give a beer to that user."
+	if (user == 'tux'):
```

Comment:
Whoops - that's specific for the 260/telephreak IRC server - tux is a 19 year old op who is lambasted for not being able to drink. Can update & remove if desired.